internal/controllers/provider/install/rbac_init.go

```diff
@@ -39,6 +39,16 @@ func newInitClusterRoleMutator(values *Values) resources.Mutator[*rbac.ClusterRo
 				Resources: []string{"customresourcedefinitions"},
 				Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
 			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"secrets"},
+				Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
```

Comment:
I think get, list, watch should be sufficient.
It only needs to read the secrets from the AccessRequest

Reply:
I changed this.